bundler/spec/plugins/install_spec.rb

```diff
@@ -263,6 +263,156 @@ def exec(command, args)
       expect(out).to include("Bundle complete!")
     end
 
+    it "installs plugins in included groups" do
+      gemfile <<-G
+        source '#{file_uri_for(gem_repo2)}'
+        group :development do
+          plugin 'foo'
+        end
+        gem 'rack', "1.0.0"
+      G
+
+      bundle "install"
+
+      expect(out).to include("Installed plugin foo")
+
+      expect(out).to include("Bundle complete!")
+
+      expect(the_bundle).to include_gems("rack 1.0.0")
+      plugin_should_be_installed("foo")
+    end
+
+    it "does not install plugins in excluded groups" do
+      gemfile <<-G
+        source '#{file_uri_for(gem_repo2)}'
+        group :development do
+          plugin 'foo'
+        end
+        gem 'rack', "1.0.0"
+      G
+
+      bundle "install --without development"
+
+      expect(out).not_to include("Installed plugin foo")
+
+      expect(out).to include("Bundle complete!")
+
+      expect(the_bundle).to include_gems("rack 1.0.0")
+      plugin_should_not_be_installed("foo")
+    end
+
+    it "upgrade plugins version listed in gemfile" do
```

Comment:
Adding a review comment here so we don't forget when we work on this PR: we should add @dfop02 attribution to the commit adding these specs, since he is the original author.

Reply:
I'm happy to. Are you just wanting a comment above the specs listing him as the original author?

Reply:
What I normally do is:

* Clone the PR including the commits I want to include or give attribution for.
* Either cherry-pick the commit directly to my PR if possible, or otherwise find name and email of original author in those commits, and then amend the relevant commit in my PR appending `Co-authored-by: name <name@example.com>` preceded by two blank lines to the commit message. See https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors.